registry/handlers/app.go

```diff
@@ -640,6 +640,7 @@ func (app *App) dispatcher(dispatch dispatchFunc) http.Handler {
 		}
 
 		context := app.context(w, r)
+		defer defaultContextManager.release(context)
```

Comment:
Is the `*http.Request` always different than the request object from `(*App).ServerHTTP`?